pkg/extractor/extractor.go

```diff
@@ -101,7 +101,7 @@ func extractYamlConfigurations(content string) (*[]Configuration, error) {
 			}
 		}
 
-		yamlByteArray, err := yaml.Marshal(anyTypeStore)
```

Comment:
where is the import?

Reply:
`yamlConvertor` is already imported previously